rootfs/init.sh

```diff
@@ -3,4 +3,8 @@
   find /cloudposse/jenkins/plugins.d/ -type f -exec cat {} \; | \
   xargs /usr/local/bin/install-plugins.sh
 
+[ -d /cloudposse/jenkins/jobs.d/ ] && \
+  ls  /cloudposse/jenkins/jobs.d | xargs -I {} mkdir -p /var/jenkins_home/jobs/{}  && \
```

Comment:
can we maybe just use just use `cp -an /cloudposse/jenkins/jobs.d/ /var/jenkins_home/jobs/`?